src/pymmcore_widgets/_util.py

```diff
@@ -143,17 +143,43 @@ def get_next_available_path(requested_path: Path | str, min_digits: int = 3) ->
     extension = requested_path.suffix
     # ome files like .ome.tiff or .ome.zarr are special,treated as a single extension
     if (stem := requested_path.stem).endswith(".ome"):
-        extension = ".ome" + extension
+        extension = f".ome{extension}"
         stem = stem[:-4]
 
+    # if ".ome.tif", remove the position index (e.g. _p0, _p1, etc.) from the stem
```

Comment:
one thing I'm not crazy about with this solution is that we're bleeding an implementation detail of `OMETiffWriter` of `pymmcore_plus.mda.handlers` into this function: specifically, the fact that you may or may not find a `_pN` suffix in the filename is entirely specific to the way that `OMETiffWriter` is (currently) implemented.  That could easily change in the future.  So, we need to determine a better way to encapsulate this information.  It *might* mean that simply using `output=save_path` isn't going to work, but I'm not sure yet.  In any case, we somehow need a more explicit way for the actual writer (which has some control over the filepath) and the widget (which also now has some control over the file path) to communicate about those paths

Reply:
That's true...what about if we simply modify the `OMETiffWriter` to create a new folder if the sequence has multiple positions and save each position in this folder? Something similar to this: https://github.com/pymmcore-plus/pymmcore-plus/pull/349.

If we do that, we do not need to update the widget...